src/components/Support/Support.module.css

```diff
@@ -0,0 +1,59 @@
+.supportContainer {
+  display: flex;
+  flex-direction: column;
+}
+.supportHeader {
+  font-family: var(--decoraction-font);
+  font-weight: 500;
+  font-size: 1.25rem;
+  line-height: 1.25;
+  color: var(--dark-text-color);
+  align-self: center;
+  margin-bottom: 16px;
+}
+.supportText {
+  font-weight: 500;
+  font-size: 1rem;
+  letter-spacing: 1.2;
+}
+.supportButtons {
+  display: flex;
+  flex-direction: column;
+  margin: 16px 0;
+  gap: 14.6px;
+}
+.supportButton {
+  font-family: var(--placeholder-font);
+  font-size: 29px;
+  line-height: 1.2;
+  font-weight: 700;
+  color: var(--deep-purple-color);
+  border: 2px solid var(--light-brand-color);
```

Comment:
na figmie border jest 2.77px ale mozna do 3 zaokraglić.
Te 2 też nie jest źle więc to taki minor do przemyślenia :D

Reply:
Zostawiłem 2, bo 3 to już wydaje się za grube